apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -351,9 +355,30 @@ public void log(Request request, Response response) {
          * the underlying Jetty classes in the response that are in classes that are not loaded in the war file so not
          * accessible to the filter.
          */
-        BfdMDC.put(
-            HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES,
-            String.valueOf(response.getHttpOutput().getWritten()));
+        Long outputSizeInBytes = response.getHttpOutput().getWritten();
+        BfdMDC.put(HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES, String.valueOf(outputSizeInBytes));
+
+        // Record the response duration.
+        Long requestStartMilliseconds =
+            (Long) request.getAttribute("http_access_response_start_milliseconds");
```

Comment:
Just a nit here; I like to put constants like this as static final String with a upper caps name.